snmp/datadog_checks/snmp/data/profiles/_cisco-wlc.yaml

```
@@ -0,0 +1,265 @@
+# Base profile for Cisco Wireless LAN Controllers
+# This profile is only compatible with the core SNMP integration
+
+device:
+  vendor: "cisco"
+
+metrics:
+  # Connected APs metrics
+
+  - MIB: AIRESPACE-WIRELESS-MIB
+    table:
+      OID: 1.3.6.1.4.1.14179.2.2.1
+      name: bsnAPTable
+    symbols:
+      - name: accessPoints
+        constant_value_one: true
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.14179.2.2.1.1.1
+          name: bsnAPDot3MacAddress
+        tag: mac_address
+      - column:
+          OID: 1.3.6.1.4.1.14179.2.2.1.1.3
+          name: bsnAPName
+        tag: ap_name
+      - column:
+          OID: 1.3.6.1.4.1.14179.2.2.1.1.4
+          name: bsnAPLocation
+        tag: ap_location
+      - column:
+          OID: 1.3.6.1.4.1.14179.2.2.1.1.19
+          name: bsnApIpAddress
+        tag: ap_ip_address
+      - column:
+          OID: 1.3.6.1.4.1.14179.2.2.1.1.6
+          name: bsnAPOperationStatus
+        mapping:
+          1: associated
+          2: disassociating
+          3: downloading
+        tag: ap_oper_status
+      - column:
+          OID: 1.3.6.1.4.1.14179.2.2.1.1.37
+          name: bsnAPAdminStatus
+        mapping:
+          1: enable
+          2: disable
+        tag: ap_admin_status
+
+  # Connected APs 802.11 interfaces metrics
+
+  - MIB: AIRESPACE-WIRELESS-MIB
+    table:
+      OID: 1.3.6.1.4.1.14179.2.2.2
+      name: bsnAPIfTable
+    symbols:
+      - name: accessPointsIf
+        OID: 1.2.3.4.5.6
+        constant_value_one: true
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.14179.2.2.2.1.1
+          name: bsnAPIfSlotId
+        tag: slot_id
+      - column:
+          OID: 1.3.6.1.4.1.14179.2.2.1.1.1
+          name: bsnAPDot3MacAddress
+        table: bsnAPTable
+        index_transform: # Keep only MAC address index
+          - start: 0
+            end: 5
+        tag: mac_address
```

```suggestion
        tag: ap_mac_address
```
for consistency with the other tags?